lib/modules/manager/github-actions/extract.ts

```diff
@@ -182,6 +184,13 @@ function extractSteps(
   deps: PackageDependency<Record<string, any>>[],
 ): void {
   for (const step of steps) {
+    if (step.uses) {
+      const val = Result.parse(step, communityActions).unwrapOrNull();
+      if (val) {
+        deps.push(val);
```

Comment:
```suggestion
        deps.push(val);
        continue;
```
I don't think we need to check further actions below?